content/guides/embed/ui-elements/open-with.md

```diff
@@ -409,3 +409,4 @@ using `openWith.addListener('execute', callback)` and
 [tools]: https://community.box.com/t5/Box-Tools/ct-p/BoxEdit
 [custom-domains]: g://embed/ui-elements/custom-domains
 [safari]: https://community.box.com/t5/Using-Box-Tools/Installing-Box-Tools/ta-p/50143
+[service-account]: https://developer.box.com/guides/authentication/user-types/
```

Comment:
Can you swap the link here from https://developer.box.com/guides/authentication/user-types/ to g://authentication/user-types/ please? It's the guide shorthand we typically use, where g:// is equivalent to https://developer.box.com/guides/